Adyen/Core/Payment Methods/Abstract/AnyPaymentMethod.swift

```diff
@@ -39,7 +39,10 @@ internal enum AnyPaymentMethod: Codable {
     case onlineBanking(OnlineBankingPaymentMethod)
     case upi(UPIPaymentMethod)
     case cashAppPay(CashAppPayPaymentMethod)
-    
+    #if canImport(TwintSDK)
```

Comment:
We potentially don't need canImport checks here, similar to Cashapp. Otherwise there will be many more canImport checks that can be avoided.